packages/ui/src/composables/useBreakpoints.ts

```diff
@@ -0,0 +1,5 @@
+import { inject } from 'vue'
+
+export const useBreakpoints = () => {
+  return inject('$vaBreakpoints')
```

Comment:
Shouldn't we use Symbol?

Reply:
It doesn't work for scheme: export this symbol in main.ts -> import it in external app (docs, for example) -> inject fails, no injection found for symbol.